app/src/main/java/org/fossasia/openevent/app/contract/model/IEventRepository.java

```diff
@@ -1,18 +1,22 @@
 package org.fossasia.openevent.app.contract.model;
 
+import org.fossasia.openevent.app.data.models.Attendee;
 import org.fossasia.openevent.app.data.models.Event;
 import org.fossasia.openevent.app.data.models.User;
 
 import java.util.List;
 
 import io.reactivex.Observable;
 
-public interface IEventModel {
+public interface IEventRepository {
```

Comment:
`EventDataRepository` suggests something like repository which contains data and can be retrieved directly from it. Hence it can be the class name implementing the model interface. 
For interface, it can be called as `DataSource`.
It is just a suggestion. :) I thought it in this way.